torch/nn/modules/distance.py

```diff
@@ -1,6 +1,39 @@
-
+import torch
+from .module import Module
 # TODO: Cosine
-# TODO: CosineDistance - make sure lua's CosineDistance isn't actually cosine similarity
+
+
+class CosineDistance(Module):
```

Comment:
Can you rename it to `CosineSimilarity`? It's not a distance - the closer the inputs, the larger the output. Distance would be `1 - similarity`.